File: task_1.js
/*
Если выписать все натуральные числа меньше 10, кратные 3 или 5, то получим 3, 5, 6 и 9. Сумма этих чисел равна 23.

Найдите сумму всех чисел меньше 1000, кратных 3 или 5.
*/

const until = 1000 //10
let arr = Array.from(Array(until - 1).keys()).map(v => v + 1) // массив от 1 до 999

let result = arr.reduce((acc, cur) => (cur % 3 === 0 || cur % 5 === 0) ? acc + cur : acc, 0)
console.log(`Result: ${result}`)

File: tasks_001-010/task_1/go/task_1.go
package main

import "fmt"

const until = 1000

func main(){
    var result int

    for num := 0; num < until; num++ {
        if num % 3 == 0 || num % 5 == 00 {
            result += num
        }
    }
	
    fmt.Println("Result:", result)
}
